app/views/report_templates/generate.html.erb

```diff
@@ -29,11 +29,12 @@
     <% end %>
   <% end %>
 
-  <%= checkbox_f f, :send_mail, label: _('Send report to e-mail') %>
-  <%= text_f f, :mail_to, {
+  <%= checkbox_f f, :send_mail,
+    label: _('Send report via e-mail'),
+    label_help: _('By checking this, the report will be sent to email address specified below. Keep unchecked if you prefer to download the report in your browser.') %>
+  <%= text_f f, :mail_to,
     label: _('Mail(s) to deliver report to'),
```

Comment:
`Mail` vs `E-mail` should be consistent on single page at least, The label probably could be "Deliver to e-mail addresses`

Reply:
Thanks :+1: